dotnet/src/webdriver/Firefox/Preferences.cs

```diff
@@ -64,6 +67,7 @@ public Preferences(Dictionary<string, object> defaultImmutablePreferences, Dicti
         /// <param name="value">A <see cref="string"/> value give the preference.</param>
         /// <remarks>If the preference already exists in the currently-set list of preferences,
         /// the value will be updated.</remarks>
+        /// <exception cref="ArgumentNullException">If <paramref name="key"/> is null.</exception>
```

Comment:
Are you writing it manually? When I see `null` here I expect it can be `<see langword="null" />`

Reply:
Yes I'm doing it manually, I don't know if Rider has special tooling but Visual Studio doesn't do much for this. Using the tag is better, let me make that change